deployment/vagrant/core.rb

```diff
@@ -29,6 +29,7 @@ def provider_libvirt(config)
 
 def provider_virtualbox(config)
   config.vm.provider :virtualbox do |vb, override|
+    vb.name = "TechEmpower Framework Benchmarks"
```

Comment:
Please leave this out. this creates conflict when creating two boxes

Reply:
OK

Reply:
Yeah I understand it is not ideal, but you can rename it after it's up if you'd like, or just create your own vagrant deployment directory

Reply:
Cancel the first CI test

Reply:
You can't tell me what to do!

Reply:
Sorry